tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -167,6 +169,22 @@
 # It is quite clear that the MRI surfaces (head, brain) are not well aligned
 # to the head digitization points (dots).
 #
+# Compute the transform
+# ---------------------
+# Let's step through the process that the transform is accomplishing.
+
+head_space = np.array([dig['r'] for dig in
+                       raw.info['dig']], dtype=float)
+
+# Transform digitization points into MRI space, adjust units by * 1e3 mm -> m
+mri_space = mne.transforms.apply_trans(trans, head_space, move=True) * 1e3
+
+# Get landmarks in voxel space, using the T1 data
+vox2ras_tkr = t1w.header.get_vox2ras_tkr()
+ras2vox_tkr = np.linalg.inv(vox2ras_tkr)
+mri_voxel_space = mne.transforms.apply_trans(ras2vox_tkr, mri_space)  # in vox
```

Comment:
I feel this should be clarified in the diagram using `graphviz`. See Figure 5.1 in [MNE-C manual](https://mne.tools/mne-c-manual/MNE-manual-2.7.3.pdf)